doc/lua_api.md

```diff
@@ -11368,6 +11368,16 @@ Functions: bit.tobit, bit.tohex, bit.bnot, bit.band, bit.bor, bit.bxor, bit.lshi
 
 See http://bitop.luajit.org/ for advanced information.
 
+Tracy Profiler
+--------------
+
+Minetest can be built with support for the Tracy profiler, which can also be
+useful for profiling mods.
+
+See doc/developing/misc.md for details.
+
+Note: This feature may be removed at any point.
```

Comment:
```suggestion
Minetest can be built with support for the Tracy profiler, which can also be
useful for profiling mods and is exposed to Lua.

Note: This is a development feature and not covered by compatibility promises.
```

Reply:
I'd like to keep the mention of the doc document.

```suggestion
Minetest can be built with support for the Tracy profiler, which can also be
useful for profiling mods and is exposed to Lua as the global `tracy`.

See doc/developing/misc.md for details.

Note: This is a development feature and not covered by compatibility promises.
```